lib/ansible/module_utils/vmware.py

```diff
@@ -127,7 +242,10 @@ def find_datastore_by_name(content, datastore_name):
 
 def find_dvs_by_name(content, switch_name):
 
-    vmware_distributed_switches = get_all_objs(content, [vim.dvs.VmwareDistributedVirtualSwitch])
+    vmware_distributed_switches = get_all_objs(
+        content, [vim.dvs.VmwareDistributedVirtualSwitch]
+    )
```

Comment:
This should not be necessary for PEP8. We have a line-length of 160, not the default 80.